Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -484,8 +607,41 @@ public Apis.Bigquery.v2.Data.Job DoCopy()
         /// </summary>
         public Apis.Bigquery.v2.Data.Job DoLoad()
         {
-            throw new NotImplementedException(
-                "Load jobs are not implemented yet.  Use Set-BqTabledata instead.");
+            if (ShouldProcess($"{Destination.TableId} (LOAD)"))
+            {
+                try
+                {
+                    var loadjob = new Apis.Bigquery.v2.Data.Job()
```

Comment:
Got it.  (fixed for the whole file, and I'll add that to my list to fix elsewhere)